admin_app/src/app/content/edit/page.tsx

```diff
@@ -68,7 +85,12 @@ const AddEditContentPage = () => {
   return (
     <FullAccessComponent>
       <Layout.FlexBox flexDirection={"column"} sx={{ p: sizes.doubleBaseGap }}>
-        <Header content_id={content_id} />
+        <Header
+          content_id={content_id}
+          onBack={() =>
+            isSaved ? router.push("/content") : setOpenDiscardChangesModal(true)
```

Comment:
I noticed something: When we make changes then discard them (whether it is adding a new tag or update the text), it still shows the modal. That can be confusing because then users would think they made changes even tho they didn't. I think we should find a way to make sure that when they content is at his original state after making changes, we can go leave the edit page.

Reply:
Good point! We just track if anything changed. Instead we may want to do a diff between what was saved and new state. This is a good amount of logic to add and will park it for another day.